cmd/admin-rpc-server.go

```diff
@@ -82,6 +82,16 @@ func (receiver *adminRPCReceiver) WriteTmpConfig(args *WriteConfigArgs, reply *V
 	return receiver.local.WriteTmpConfig(args.TmpFileName, args.Buf)
 }
 
+// BackupConfigArgs - save a copy of the current config file
+type BackupConfigArgs struct {
+	AuthArgs
+}
+
+// BackupConfig - save a copy of the current config file
+func (receiver *adminRPCReceiver) BackupConfig(args *CommitConfigArgs, reply *VoidReply) error {
```

Comment:
Should the args be of type `*BackupConfigArgs`

Reply:
> Should the args be of type *BackupConfigArgs

Fixed